apps/meteor/client/views/e2e/SaveE2EPasswordModal.tsx

```diff
@@ -26,13 +29,21 @@ const SaveE2EPasswordModal = ({ randomPassword, onClose, onCancel, onConfirm }:
 			confirmText={t('I_Saved_My_Password')}
 			variant='warning'
 			title={t('Save_your_encryption_password')}
+			annotation={t('You_can_do_from_account_preferences')}
 		>
-			<>
-				<Box dangerouslySetInnerHTML={{ __html: t('E2E_password_reveal_text', { randomPassword }) }} />
-				<Button disabled={hasCopied} small mbs={24} onClick={() => copy()}>
-					{hasCopied ? t('Copied') : t('Copy_password')}
-				</Button>
-			</>
+			<span>
```

Comment:
It should be a block level element instead, as it have `div`s (from `Box` and `CodeSnippet`) as siblings.